config/7.10.0/common.yaml

```diff
@@ -678,12 +681,14 @@ servers:
                       value: "jboss"
                     - name: HTTPS_PASSWORD
                       value: "[[$.KeyStorePassword]]"
+                    - name: KUBERNETES_NAMESPACE
+                      valueFrom:
+                        fieldRef:
+                          fieldPath: metadata.namespace
+                    - name: KUBERNETES_LABELS
+                      value: "cluster=kie-cluster"
```

Comment:
does this value work well when multiple kie-server objects are defined? does each DeploymentConfig get its own cluster?

e.g. 
```yaml
spec:
  objects:
    servers:
      - name: server1
      - name: server2
      - name: server3
```

Reply:
Every deployment config use its cluster label to be visible by the jgroups and export read the env with the same value ie. kubernetes.ping.kieserver-cluster, kubernetes.ping.console-cluster, kubernetes.ping.consolemon-cluster here is more visible https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/504/files

Reply:
sure, but we need to be sure it's a unique name to each kie server DC (of which there may be multiple per kieapp)... AND unique to each kieapp (of which there may be multiple per namespace)

maybe leverage the existing `[[.KieName]]` var somehow?

Reply:
jgrp.k8s.console.[[.KieName]]   jgrp.k8s.kieserver.[[.KieName]]  jgrp.k8s.dashbuilder.[[.KieName]]